test/cpp/test_ir.cpp

```diff
@@ -46,7 +49,7 @@ TEST(IrTest, TestHash) {
   EXPECT_NE(add1->hash(), sub->hash());
 }
 
-TEST(IrTest, TestSelectUnselect) {
+TEST_F(IrTest, TestSelectUnselect) {
   ForEachDevice([&](const torch::lazy::BackendDevice& device) {
```

Comment:
Wondering if the "-> void" is required

Reply:
sorry what do you mean?

Reply:
Sorry never mind, I may have looked at a staled commit.